statistics/handle/bootstrap.go

```diff
@@ -130,23 +135,26 @@ func (h *Handle) initStatsHistograms4Chunk(is infoschema.InfoSchema, cache *stat
 				continue
 			}
 			hist := statistics.NewHistogram(id, ndv, nullCount, version, &colInfo.FieldType, 0, totColSize)
-			hist.Correlation = row.GetFloat64(9)
+			hist.Correlation = row.GetFloat64(8)
 			col := &statistics.Column{
 				Histogram:  *hist,
 				PhysicalID: table.PhysicalID,
 				Info:       colInfo,
 				Count:      nullCount,
 				IsHandle:   tbl.Meta().PKIsHandle && mysql.HasPriKeyFlag(colInfo.Flag),
-				Flag:       row.GetInt64(10),
+				Flag:       row.GetInt64(9),
 			}
 			lastAnalyzePos.Copy(&col.LastAnalyzePos)
 			table.Columns[hist.ID] = col
 		}
 	}
 }
 
-func (h *Handle) initStatsHistograms(is infoschema.InfoSchema, cache *statsCache) error {
-	sql := "select HIGH_PRIORITY table_id, is_index, hist_id, distinct_count, version, null_count, cm_sketch, tot_col_size, stats_ver, correlation, flag, last_analyze_pos from mysql.stats_histograms"
+// load ALL the meta data without cm_sketch
```

Comment:
comment of function header should begin with function name.
`initStatsHistograms loads ...`